src/topology/algebra/group.lean

```diff
@@ -501,6 +501,39 @@ lemma topological_group.of_comm_of_nhds_one {G : Type u} [comm_group G] [topolog
   (hleft : ∀ x₀ : G, 𝓝 x₀ = map (λ x, x₀*x) (𝓝 1)) : topological_group G :=
 topological_group.of_nhds_one hmul hinv hleft (by simpa using tendsto_id)
 
+/-- A homomorphism of topological groups is continuous if and only if it is continuous at 1. -/
+@[to_additive "A homomorphism of topological additive groups is continuous if and only if it is
+continuous at 0."]
+lemma topological_group.continuous_iff_continuous_at_one [topological_space H] [group H]
+  [topological_group H] {f : G →* H} : continuous f ↔ continuous_at f 1 :=
```

Comment:
Could you please formulate this for `{F : Type*} [monoid_hom_class F G H] {f : F}`? Then we'll be able to use it for `linear_map`s etc without explicit coercion to monoid homomorphisms.

Reply:
UPD: I'll do this and golf the next proof.

Reply:
@urkud Are you still planning to do this? Otherwise I can do it.